src/vehicle.cpp

```diff
@@ -616,6 +616,12 @@ bool vehicle::is_engine_type( const int e, const itype_id  &ft ) const
     return part_info( engines[e] ).fuel_type == ft;
 }
 
+bool vehicle::is_perpetual_type( const int e ) const
+{
+    const itype_id  &ft = part_info( engines[e] ).fuel_type;
```

Comment:
Excessive space before `&ft`.

Reply:
I see. I should fix that.

Reply:
Okay, now I don't see.
vortex engines are TOOLS, they don't have a tank capacity or ammo count. So they're not a valid fuel source for get_printable_fuel_types(), and don't display a fuel indicator,
 like so:
![vortexenginefuelindicators](https://user-images.githubusercontent.com/8007405/44950354-4a58b180-ae0b-11e8-92b5-eb57b1930a57.png)

vortex generators are a little weird, because they show up in the Reactors list, but I don't think that's a problem.
![vortexgeneratorfuelindicators](https://user-images.githubusercontent.com/8007405/44950387-277acd00-ae0c-11e8-8d25-7f80f5ff3fcd.png)